Automattic/jetpack
# Nav Unification: Expose My Home menu to all roles

Partially fixes https://github.com/Automattic/jetpack/issues/19506.

#### Changes proposed in this Pull Request:
We're enabling My Home to all WP.com users in https://github.com/Automattic/wp-calypso/pull/58910 and D71149-code, so this PR ensures they can access it via the menu.

#### Jetpack product discussion
N/A.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
- Prepare your test environment:
  - Simple sites: Apply D71199-code and sandbox the API.
  - Atomic sites: Install Jetpack Beta and switch to the `update/nav-unification-my-home-all-roles` branch.
- Make sure My Home is visible in the menu for all roles (Administrators, Authors, Editors, and Contributors).
